.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,31 @@
+name: Upgrade Python Requirements
+
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch against which to create requirements PR"
+        required: true
+        # If copying this template manually, you must provide your default branch name
+        # in quotes, such as 'master'
+        default: "main"
+
+jobs:
+  call-upgrade-python-requirements-workflow:
+    uses: openedx/.github/.github/workflows/upgrade-python-requirements.yml@master
+    with:
+    # If copying manually, also provide your default branch name in quotes here
+      branch: ${{ github.event.inputs.branch || 'main' }}
+      # optional parameters below; fill in if you'd like github or email notifications
+      # user_reviewers: ""
+      team_reviewers: "axim-aximprovements"
+      email_address: "aximimprovements@axim.org"
+      # send_success_notification: false
```

Comment:
We can keep send_success_notification to false to avoid weekly success emails from many repositories.

Reply:
Done